### PR TITLE
Added default index page, default startPage, removed iframe

### LINKED ISSUE #237: Product Onboarding
We need to improve an experience of the first run

### PR DESCRIPTION
**Pull request by slaveeks**
resolves #237 #194 #232

### CODE REVIEW

**Comment by talyguryn** [comment]
#197 ?

File: src/backend/views/pages/index.twig [hunk 1]
```diff
@@ -14,11 +14,13 @@
...
-  <div class="landing-loader" id="frame-loader">
-    {{ svg('loader') }}
+  <div class="landing-content">
+    {{ svg('frog') }}
+    <p>
+      It’s time to create the first page!
+    </p>
+    {% include 'components/button.twig' with {label: 'Add page', icon: 'plus', size: 'small', url: '/page/new'} %}
   </div>
-  <iframe class="landing-frame" src="{{ config.landingFrameSrc }}" seamless frameborder="0" onload="this.style.opacity = 1; setTimeout(document.getElementById('frame-loader').remove(), 500)"></iframe>
```
Comment (neSpecc) [inline src/backend/views/pages/index.twig:20]: тут в 6 строке еще используется `landingFrameSrc` как минимум

File: src/backend/views/pages/index.twig [hunk 2]
```diff
@@ -14,11 +14,13 @@
 </script>
 <body class="landing-body">
   {% include "components/header.twig" %}
-  <div class="landing-loader" id="frame-loader">
-    {{ svg('loader') }}
+  <div class="landing-content">
+    {{ svg('frog') }}
+    <p>
```
Comment (neSpecc) [inline src/backend/views/pages/index.twig:19]: а где отступы по 26px вокруг `<p>` ?

**Comment by slaveeks** [comment]
> #197 ?

Oh, sorry, I did not. Why hasn't this pr been merged yet?

File: src/backend/views/pages/index.twig [hunk 3]
```diff
@@ -14,11 +13,13 @@
 </script>
 <body class="landing-body">
   {% include "components/header.twig" %}
-  <div class="landing-loader" id="frame-loader">
-    {{ svg('loader') }}
+  <div class="landing-content">
```
Comment (TatianaFomina) [inline src/backend/views/pages/index.twig:16]: now that there's no landing inserted via iframe, maybe we should use different class names?

**Review by TatianaFomina — APPROVED** [review]

**Review by robonetphy — APPROVED** [review]

**Pull request merged**